src/setupApiClient.ts

```diff
@@ -12,7 +12,7 @@ const {
   MAILSCRIPT_API_SERVER = undefined,
 } = process.env
 
-async function setupApiClient(): Promise<typeof api> {
+async function setupApiClient(): typeof api {
```

Comment:
> I don't like TypeScript
I hear you, we've all been there.

Code blocks wrapped in `async/await` will always return a function.
This would be the same as if you were doing the following:

```
function exportedFn(): Promise<typeof ...> {
  return new Promise(() => {
    // code
  })
}
```